modules/displayItem.js

```diff
@@ -24,13 +23,23 @@ export const addTask = () => {
       completed: false,
       index: arrayOfTasks.length + 1,
     };
-
     arrayOfTasks.push(task);
     localStorage.setItem('arrayOfTasks', JSON.stringify(arrayOfTasks));
     displayTasks();
   }
 };
 
+// clearAll
+export const clearAll = () => {
+  arrayOfTasks = arrayOfTasks.filter((abc) => abc.completed === false);
+  arrayOfTasks = arrayOfTasks.map((task, index) => {
+    task.index = index + 1;
+    return task;
+  });
+  localStorage.setItem('arrayOfTasks', JSON.stringify(arrayOfTasks));
+  displayTasks();
+};
```

Comment:
- [ ] Based on the project requirement, completed Items are cleared when the `clear all completed` button is clicked,  the task object is updated in the local storage as expected but the only challenge I see is when the page is refreshed, the items marked completed does not appear on the UI with a strikethrough as completed. Please see the image below 👇🏾 

![image](https://github.com/SanaTameem/To-do-List/assets/84156714/6609596e-b082-4a32-adee-3a0a04c4ba85)

Please ensure that when the page is refreshed the true state of the task object updates the UI correctly. If task `A` was marked as completed, when we refreshed, on the UI that task should have a strikethrough and the checkbox marked. .